addons/survey/models/survey_user.py

```diff
@@ -58,7 +59,7 @@ class SurveyUserInput(models.Model):
     question_ids = fields.Many2many('survey.question', string='Predefined Questions', readonly=True)
     deadline = fields.Datetime('Deadline', help="Datetime until customer can open the survey and submit answers")
 
-    quizz_score = fields.Float("Score for the quiz (%)", compute="_compute_quizz_score", default=0.0)
+    quizz_score = fields.Float("Score (%)", compute="_compute_quizz_score", default=0.0)
```

Comment:
While we are at it let us remove the default, computed fields should not have default :) .

Reply:
It's cleaned.